pkg/storage/utils/filelocks/filelocks.go

```diff
@@ -28,7 +28,13 @@ import (
 	"github.com/gofrs/flock"
 )
 
-var _localLocks sync.Map
+var (
+	localLocks sync.Map
```

Comment:
please undo this change. unexported global vars should be prefixed with `_` so they can be easily identified